proto/uber/cadence/api/v1/service_visibility.proto

```diff
@@ -66,6 +69,23 @@ message ListWorkflowExecutionsResponse {
   bytes next_page_token = 2;
 }
 
+message ListAllWorkflowExecutionsRequest {
+  string domain = 1;
+  int32 maximum_page_size = 2;
+  bytes next_page_token = 3;
+  StartTimeFilter start_time_filter = 4;
+  bool partial_match = 5;
+  repeated StatusFilter close_status = 6;
+  string workflow_search_value = 7;
```

Comment:
what is this field? would be nice to add one line comment on each of these fields in the request